data/campaigns/tutorial02_warfare.wmf/scripting/texts.lua

```diff
@@ -126,6 +126,54 @@ trainingcamp2 = {
    )
 }
 
+scouting1 = {
+   position = "topright",
+   title = _"Scout’s Hut",
+   body = (
+      h1(_"Explore Your Surroundings") ..
+      p(_[[Large parts of the map haven’t been explored yet. It might be a good idea to know which direction is best for expanding our empire.]]) ..
+      p(_[[You can try your luck and expand randomly, but it is risky. A safer way is to use a scout to explore unseen parts of the map. What if he finds mountains overflowing with gold?]]) ..
+      p(_[[We will need a scout’s hut to start exploring. It is a small building.]]) ..
+      li_object("barbarians_scouts_hut", _[[Build a scout’s hut.]], plr.color)
+   ),
+   h = 300,
+   obj_name = "build_scouts_hut",
+   obj_title = _"Build a scout’s hut",
+   obj_body = (
+      p(_[[It’s good to know your surroundings. A scout will explore unseen parts of the map for you.]]) ..
+      li(_[[Build a Scout’s Hut.]])
+   )
+}
+
+scouting2 = {
+   position = "topright",
+   title = _"Scout is ready",
+   body = (
+      h1(_"Ready to go!") ..
+      p(_[[Well done, your scout’s hut has been built. While our scout is moving in, let’s learn a little bit more about him.]]) ..
+      p(_[[First of all, he needs to take some food for his scouting trips. By chance, we have some rations in our warehouse.]]) ..
+      li_object("barbarians_scout", _[[When provided with food, he will walk randomly around the area. You should follow his movement before fog of war hides what he saw.]], plr.color)
```

Comment:
I have another idea: This would be an opportunity to teach the watch window.

Reply:
Like asking a player to set watch window on the scout? I don't know if it is possible.

Reply:
That's exactly what I mean :)

The watch window is called "watch", you can check if that has been opened. It has a button "follow". You could check if that was clicked, but not if the window is actually following the scout. Checking the button gets complicated a bit if the player opens multiple watch window.

If you want to do this, let's to it in a separate branch.

Reply:
@gunchleoc I was unable to find any way how to set watch window on a scout. How do I do that?

Reply:
The tricky bit is that you have to catch him.

The way I did it was:

* Pause the game when he was near a flag location
* Click on the map and open the watch window.
* Hit the follow button.
* Unpause the game

Alternatively, you can also watch the flag in front of the building and when he comes out or returns, hit the follow button.